helm/charts/oathkeeper-maester/README.md

```diff
@@ -8,45 +8,46 @@ A Helm chart for deploying ORY Oathkeeper Rule Controller in Kubernetes
 
 ## Maintainers
 
-| Name | Email | Url |
-| ---- | ------ | --- |
+| Name     | Email         | Url                 |
```

Comment:
Those files are auto-generated by helm-docs plugin from our values.yaml. The `make format` and the plugin will override each other all the time 😞